pytorch_transformers/tokenization_utils.py

```diff
@@ -589,16 +590,30 @@ def add_special_tokens(self, special_tokens_dict):
 
         return added_tokens
 
-    def tokenize(self, text, **kwargs):
+    def tokenize(self, text, split_additional_on_word_boundaries=False, **kwargs):
```

Comment:
Seems good to m. I think this argument could be renamed `additional_tokens_as_full_words_only`

Reply:
Thank @thomwolf for suggesting a good name! I've renamed that variable 😃